packages/dashboard/src/managers/rmf-health-state-manager.ts

```diff
@@ -0,0 +1,74 @@
+import { DoorSummary, LiftSummary, DispenserSummary, RobotSummary } from 'react-components';
+
+export interface HealthStatus {
+  door: DoorSummary;
+  lift: LiftSummary;
+  dispenser: DispenserSummary;
+  robot: RobotSummary;
+}
+
+export default class RmfHealthStateManager {
+  getDoorSummary = (): DoorSummary => {
+    return {
+      item: 'Door',
+      itemSummary: { operational: 2, outOfOrder: 1 },
```

Comment:
I think this will cause inconsistency if  using any maps that does not have exactly 3 doors, this is not easily fixed so I might suggest to copy the new `main-menu.tsx` into a temporary `wip-main-menu-plusplus.tsx` (or similar) and revert the main menu changes.

In other words, this will be a new feature that is unused and disabled by default, the downside is that you need to ensure this temp main-menu is keep up to date with other main menu changes until it can be merged into it (when the backend is ready).